plotting_rbfes_with_cinnabar/PlottingFreeEnergiesUsingCinnabar.ipynb

```diff
@@ -360,66 +377,66 @@
+      "                'calculated_estimate': 0.530999999999999,\n",
       "                'experimental_error': 0.18,\n",
       "                'experimental_estimate': -9.81},\n",
-      " 'lig_ejm_43': {'calculated_error': 0.18,\n",
-      "                'calculated_estimate': -8.29,\n",
+      " 'lig_ejm_43': {'calculated_error': 0.12599206324209478,\n",
+      "                'calculated_estimate': 1.730999999999999,\n",
       "                'experimental_error': 0.18,\n",
       "                'experimental_estimate': -8.29},\n",
-      " 'lig_ejm_46': {'calculated_error': 0.17,\n",
-      "                'calculated_estimate': -11.35,\n",
+      " 'lig_ejm_46': {'calculated_error': 0.07080960386840184,\n",
+      "                'calculated_estimate': -0.6889999999999962,\n",
       "                'experimental_error': 0.17,\n",
       "                'experimental_estimate': -11.35},\n",
-      " 'lig_ejm_47': {'calculated_error': 0.18,\n",
-      "                'calculated_estimate': -9.73,\n",
+      " 'lig_ejm_47': {'calculated_error': 0.18712028217165563,\n",
+      "                'calculated_estimate': 0.040999999999998926,\n",
       "                'experimental_error': 0.18,\n",
       "                'experimental_estimate': -9.73},\n",
-      " 'lig_ejm_48': {'calculated_error': 0.18,\n",
-      "                'calculated_estimate': -9.03,\n",
+      " 'lig_ejm_48': {'calculated_error': 0.1871202821716555,\n",
+      "                'calculated_estimate': 0.5409999999999974,\n",
       "                'experimental_error': 0.18,\n",
       "                'experimental_estimate': -9.03},\n",
-      " 'lig_ejm_50': {'calculated_error': 0.18,\n",
-      "                'calculated_estimate': -9.01,\n",
+      " 'lig_ejm_50': {'calculated_error': 0.08327064308626422,\n",
+      "                'calculated_estimate': 0.9810000000000002,\n",
       "                'experimental_error': 0.18,\n",
       "                'experimental_estimate': -9.01},\n",
-      " 'lig_jmc_23': {'calculated_error': 0.18,\n",
-      "                'calculated_estimate': -11.74,\n",
+      " 'lig_jmc_23': {'calculated_error': 0.08374962686484035,\n",
+      "                'calculated_estimate': -1.0789999999999966,\n",
       "                'experimental_error': 0.18,\n",
       "                'experimental_estimate': -11.74},\n",
-      " 'lig_jmc_27': {'calculated_error': 0.17,\n",
-      "                'calculated_estimate': -11.31,\n",
+      " 'lig_jmc_27': {'calculated_error': 0.11407892005098923,\n",
+      "                'calculated_estimate': -1.288999999999997,\n",
       "                'experimental_error': 0.17,\n",
       "                'experimental_estimate': -11.31},\n",
-      " 'lig_jmc_28': {'calculated_error': 0.18,\n",
-      "                'calculated_estimate': -11.01,\n",
+      " 'lig_jmc_28': {'calculated_error': 0.07933473388119473,\n",
```

Comment:
Could you do a 2d.p. truncation on the results writing so calculated & experimental are consistent? (lines 452-453 & 463-464)